x-pack/test/idp-fixture/docker-compose.yml

```diff
@@ -38,3 +38,15 @@ services:
       - ./idp/shibboleth-idp/conf:/opt/shibboleth-idp/conf
       - ./idp/shibboleth-idp/credentials:/opt/shibboleth-idp/credentials
       - ./idp/shib-jetty-base/start.d/ssl.ini:/opt/shib-jetty-base/start.d/ssl.ini
+
+  oidc-provider:
+    image: "c2id/c2id-server:7.8"
+    depends_on:
+      - openldap
+    ports:
+      - "8080:8080"
```

Comment:
can we avoid the hardcoded port? See #40333 for my suggestion on how to do it for the rest of the fixture.

Reply:
yes! Will do :) I wasn't aware this could be handled this way, thanks!